sematic/logging.py

```diff
@@ -51,7 +57,7 @@ def make_log_config(log_to_disk: bool = False):
         "loggers": {
             "sematic": root_logger_config,
             "gunicorn.error": {"level": "ERROR", "handlers": full_handler_list},
-            "gunicorn.access": {"level": "INFO", "handlers": full_handler_list},
+            "gunicorn.access": {"level": level, "handlers": full_handler_list},
```

Comment:
since `level` is now an int, the error sub-dict will have "ERROR" and the this one will have an int.  doesn't look right?

Reply:
the logs API accepts either, actually. Though I can change the error one to use `logging.ERROR` so that it's more consistent in appearance.